src/locale/locales/fr/messages.po

```diff
@@ -5677,6 +5701,14 @@ msgstr "Partager quand même"
 msgid "Share feed"
 msgstr "Partager le fil d’actu"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:621
+msgid "Share image externally"
+msgstr "Partager l’image avec d’autres personnes"
```

Comment:
Well, yeah, but, _ailleurs_ is shorter and works well in that specific context imho.

```suggestion
msgstr "Partager l’image ailleurs"
```